### PR TITLE
Enhance clustering protocols

### LINKED ISSUE #438: Add standard deviation to `clustfcc.txt`

### LINKED ISSUE #453: `seletopclusts` should dump the selection to its folder

### PR DESCRIPTION
**Pull request by rvhonorato**
You are about to submit a new Pull Request. Before continuing make sure you read the [contributing guidelines](CONTRIBUTING.md) and you comply with the following criteria:

- [X] You have stick to Python. Talk with us before adding other programming languages to HADDOCK3
- [ ] Your PR is about CNS
- [ ] Your code is well documented: proper docstrings and explanatory comments for those tricky parts
- [ ] You structured the code into small functions as much as possible. You can use classes if there's a (state) purpose
- [X] code follows our coding style
- [ ] You wrote tests for the new code
- [X] `tox` tests pass. *Run `tox` command inside the repository folder*
- [ ] `-test.cfg` examples execute without errors. *Inside `examples/` run `python run_tests.py -b`*
- [X] PR does not add any *install dependencies* unless permission granted by the HADDOCK team
- [X] PR does not break licensing
- [ ] Your PR is about writing documentation for already existing code :fire:
- [ ] Your PR is about writing tests for already existing code :godmode:

---

Improves the clustering to make it more useful, closes #453 and closes #438

### CODE REVIEW

**Comment by joaomcteixeira** [comment]
I am running the `*-test.cfg` to see if all goes all. manual integration tests :-o

File: src/haddock/modules/analysis/seletopclusts/__init__.py [hunk 1]
```diff
@@ -59,10 +59,15 @@ def _run(self):
                                 and pdb.clt_model_rank == model_ranking
                                 ):
                             self.log(
-                                f"Selecting {pdb.file_name} "
-                                f"Cluster {target_ranking} "
-                                f"Model {model_ranking}"
+                                f" {pdb.file_name} "
```
Comment (mgiulini) [inline src/haddock/modules/analysis/seletopclusts/__init__.py:62]:
```suggestion
                                f"{pdb.file_name} "
```
Better not to have two spaces here right?

**Review by mgiulini — APPROVED** [review]
great!
I would maybe add a comment in `src/haddock/modules/analysis/clustfcc/__init__.py` to specify that we are extracting statistical quantities from the clusters (for the unaware future developer)

**Review by joaomcteixeira — APPROVED** [review]
The `*-test.cfg` workflows are stable. :+1:

**Comment by rvhonorato** [comment]
> great! I would maybe add a comment in `src/haddock/modules/analysis/clustfcc/__init__.py` to specify that we are extracting statistical quantities from the clusters (for the unaware future developer)

True, this can go when we do #387 tho, the module needs some extra doc anyway

**Pull request merged**